.github/workflows/docs.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         with:
           python-version: 3.7
 
-      - run: sudo npm install katex@latest -g
+      - run: sudo npm install katex@0.13.0 -g
```

Comment:
Can't we use the latest, what is the reason ?

Reply:
I am afraid it breaks the rendering.

Reply:
Yes, this weird...